core/src/mindustry/logic/LAssembler.java

```diff
@@ -93,6 +94,12 @@ public LVar var(String symbol){
         if(symbol.startsWith("0x")) return Strings.parseLong(symbol, 16, 2, symbol.length(), invalidNum);
         if(symbol.startsWith("+0x")) return Strings.parseLong(symbol, 16, 3, symbol.length(), invalidNum);
         if(symbol.startsWith("-0x")) return -Strings.parseLong(symbol, 16, 3, symbol.length(), invalidNum);//FIXME: breaks with Long.MIN_VALUE
+        if(symbol.startsWith("%[") && symbol.endsWith("]") && symbol.length() > 3) {
+            double color = parseNamedColor(symbol);
+            if (color != -1d) {
```

Comment:
Incorrect formatting, extra spaces. Same issue in the other if-statement.

Reply:
should be fixed in https://github.com/Anuken/Mindustry/pull/10682/commits/2c6319e0f1039f920ce776749fc93a2e246a3ebd